lib/common/components/forms/form_component.dart

```diff
@@ -64,16 +67,20 @@ class _CustomFormState extends State<CustomForm> {
           if (widget.hasForgotPassword)
             Row(
               mainAxisAlignment: MainAxisAlignment.end,
-              children: const [
-                CustomTextButton(text: 'Forgot Password?', type: 'inverse'),
+              children: [
+                CustomTextButton(
+                  text: 'Forgot Password?',
+                  type: TextButtonType.tertiary,
+                  onPressedHandler: widget.textButtonHandler,
```

Comment:
I think you need to change the onPressedHandler for the forgot password, since the handler is different from the textButton at the bottom of the page, maybe you can just straight up put the navigator there